app/services/word_watcher.rb

```diff
@@ -130,10 +130,10 @@ def self.censor_text(text)
     regexps.inject(text) { |txt, regexp| censor_text_with_regexp(txt, regexp) }
   end
 
-  def self.replace_text(text)
+  def self.replace_text(text, watch_word_type: %i[replace link])
```

Comment:
Initially thought about extracting two methods, one for replace and one for link, but the methods would look too similar, so I promoted the variable into a parameter instead.

Reply:
This looks good to me. I think I also wanted to have different methods because I did not like having that loop in the function.

Anyway, the naming in this file is not very good, unfortunately. We should probably have "replace_text" and "replace_link" as types, at least to start.

Also judging by the name, `replace_text` and `censor_text` should work on text AND return text -> that means they should not return any HTML. Also, `censor` should probably be named `censor_html`. Or even better, we could merge the two methods and then have `censor(text: nil, html: nil)` or something like that.